orchagent/debugcounterorch.cpp

```diff
@@ -34,13 +35,56 @@ DebugCounterOrch::DebugCounterOrch(DBConnector *db, const vector<string>& table_
 {
     SWSS_LOG_ENTER();
     publishDropCounterCapabilities();
+
+    gPortsOrch->attach(this);
 }
 
 DebugCounterOrch::~DebugCounterOrch(void)
 {
     SWSS_LOG_ENTER();
 }
 
+void DebugCounterOrch::update(SubjectType type, void *cntx)
+{
+    SWSS_LOG_ENTER();
+
+    if (type == SUBJECT_TYPE_PORT_CHANGE) {
```

Comment:
`{ `in the separate line.   Can you change this to be consistent in all the changes in the PR?

Reply:
done